portfolio/src/main/webapp/script.js

```diff
@@ -12,31 +12,87 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+window.addEventListener('DOMContentLoaded', () => {
+  const navBar = document.getElementById('nav-bar');
+
+  Array.from(navBar.children).forEach(child => {
+    child.addEventListener('click', () => {
+      const targetId = child.getAttribute('data-target-id');
+      const targetElement = document.getElementById(targetId);
+      showContent(targetElement);
+    });
+  });
+});
+
+/* Displays the content title and text in the content container */
```

Comment:
nit: Should this be "/**"?

See https://google.github.io/styleguide/jsguide.html#terminology-notes

Reply:
Yea, I fixed it.